docs/use-cases.md

```diff
@@ -0,0 +1,59 @@
+# Overview
+
+KubeVirt is a tool for scheduling and managing the lifecycle of Virtual Machines (VMs) across a distributed set of nodes.  In a broad sense KubeVirt is targeting all use-cases involving VM placement and management in the datacenter.
+
+This means KubeVirt is being built with a feature set capable of delivering both the scale out functionality that powers cloud IaaS projects as well as the high availability features desired for on premise datacenter virtualization. Since KubeVirt is built upon Kubernetes, this also introduces the option of a hybrid cluster where both container and virtualized workloads coexist.
+
+This may sound ambitious and slightly confusing considering how many other projects exist in the VM scheduling space. By comparing KubeVirt to these other projects, it should be clear what KubeVirt’s scope is and where it fits in the ecosystem.
+
+# KubeVirt vs. Other Projects
+
+## Kubernetes
+
+Kubernetes is an open source project built to automate deployment and lifecycle management of containerized applications.
+
+KubeVirt is a drop in addon that can be applied to any to Kubernetes cluster that gives Kubernetes the ability to manage VMs.
+
+## OpenStack
+
+OpenStack is an open source IaaS platform built of various components to provide a wide range of features such as compute (VM scheduling), networking, block storage, object storage, ect.
+
+KubeVirt is a single component focused entirely on VM scheduling.  Network and block storage are something that KubeVirt consumes from other systems and not something KubeVirt attempts to define. Similar to the way Libvirt has become a standard for managing VMs on a local node across multiple projects, we hope to see KubeVirt become the widely adopted standard for scheduling VMs.  It is theoretically possible OpenStack could adopt KubeVirt for VM scheduling.
+
+## Nova
+
+Nova is the open source component of OpenStack responsible for VM scheduling. Nova provides an abstraction layer that allows multiple virtualization technologies to be utilized. This means Nova is maintaining drivers for technologies such as Xen, VMware, KVM, Hyper-V, and LXC.
+
+KubeVirt has made the choice to focus on KVM managed by Libvirt. Initially this may sound like it is limiting KubeVirt’s functionality, but in practice we see KVM and Libvirt emerging as the leading standard in open source virtualization. By focusing on this technology stack we aren’t limited by creating a one size fits all virtualization technology abstraction.  We can fully support a feature set that leverages all that Libvirt and KVM have to offer.
+
+## oVirt and RHEV
```

Comment:
Just remove the reference to RHEV entirely, and talk about oVirt exclusively.   RHEV is just a particular vendor's productization of oVirt so its redundant to list it. That avoids us chasing whatever preferred product naming is this week.